src/database/queries/test/test.js

```diff
@@ -0,0 +1,23 @@
+const test = require('tape');
+const runDbBuild = require('../../db_build');
+const { getLibraryBooks } = require('../book');
+
+
+test('Test getLibraryBooks', (t) => {
+  runDbBuild('db_bulid.sql', (err, res) => {
+    t.notOk(err);
+    return runDbBuild('fake_data.sql', () => {
+      getLibraryBooks()
+        .then((response) => {
+          t.equal(response.length > 0, true, 'successfully');
```

Comment:
use `response.length === 5` (`5` as example) insted of `response.length > 0`  
you know the books number in the `fake_data` so be specific